File: app/src/main/java/workshop1024/com/xproject/model/publishertype/PublisherType.java
package workshop1024.com.xproject.model.publishertype;

/**
 * 发布者内容类型
 */
public class PublisherType {
    //单选项类型id
    private String mTypeId;
    //单选项名称
    private String mName;

    public PublisherType(String typeId, String name) {
        mTypeId = typeId;
        mName = name;
    }

    public String getTypeId() {
        return mTypeId;
    }

    public void setTypeId(String typeId) {
        mTypeId = typeId;
    }

    public String getName() {
        return mName;
    }

    public void setName(String name) {
        mName = name;
    }
}

File: main/src/main/java/workshop1024/com/xproject/main/publisher/data/PublisherType.kt
package workshop1024.com.xproject.main.publisher.data

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

/**
 * 发布者内容类型
 */
@Entity(tableName = "publishertypes")
data class PublisherType(//单选项类型id
        @PrimaryKey @ColumnInfo(name = "typeId") var mTypeId: String, //单选类型
        @ColumnInfo(name = "type")var mType: String, // 单选项名称
        @ColumnInfo(name = "name")var mName: String)